util/plugins/eu.esdihumboldt.util/src/eu/esdihumboldt/util/geometry/NumberFormatter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2016 wetransform GmbH
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     wetransform GmbH <http://www.wetransform.to>
+ */
+
+package eu.esdihumboldt.util.geometry;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.util.regex.Pattern;
+
+import de.fhg.igd.slf4jplus.ALogger;
+import de.fhg.igd.slf4jplus.ALoggerFactory;
+
+/**
+ * Format the number in required representation
+ * 
+ * @author Arun
+ */
+public class NumberFormatter {
+
+	private static final ALogger log = ALoggerFactory.getLogger(NumberFormatter.class);
+
+	/**
+	 * Format number to specified format
+	 * 
+	 * @param value double value
+	 * @param format a pattern
+	 * @return String presentation of a formatted number
+	 */
+	public static String formatTo(double value, String format) {
```

Comment:
As this is called for every single double value I think it would make sense to instead pass the `DecimalFormat` to the writers instead of the string, avoiding to create and configure the format many many times.